packages/typescript-estree/src/parseSettings/createParseSettings.ts

```diff
@@ -46,6 +48,14 @@ export function createParseSettings(
       ? tsestreeOptions.tsconfigRootDir
       : process.cwd();
   const passedLoggerFn = typeof tsestreeOptions.loggerFn === 'function';
+  const filePath = ensureAbsolutePath(
+    typeof tsestreeOptions.filePath === 'string' &&
+      tsestreeOptions.filePath !== '<input>'
```

Comment:
As I checked [`<text>` still used in ESLint](https://github.com/search?q=repo%3Aeslint%2Feslint%20%3Ctext%3E&type=code), should we ignore it too? This code exists before this PR, proberly won't cause problems. When I saw this code, I thought maybe they removed `<text>` placeholder. :)

Reply:
I thought the `<text>` placeholder was never used because [typescript-eslint provides a default value](https://typescript-eslint.io/packages/typescript-estree#parsing) for filepath. The documentation described the code as follows,

```ts
const PARSE_DEFAULT_OPTIONS: ParseOptions = {
  comment: false,
  filePath: 'estree.ts', // or 'estree.tsx', if you pass jsx: true
  jsDocParsingMode: 'all',
  jsx: false,
  loc: false,
  loggerFn: undefined,
  range: false,
  tokens: false,
};

declare function parse(
  code: string,
  options: ParseOptions = PARSE_DEFAULT_OPTIONS,
): TSESTree.Program;
```

But, here's the code I actually found

```ts
function parse<T extends TSESTreeOptions = TSESTreeOptions>(
  code: string,
  options?: T,  // there's no default value
): AST<T> {
  const { ast } = parseWithNodeMapsInternal(code, options, false);
  return ast;
}
```

Maybe it's just something missed over time. 

Should this be fixed in the current PR? I feel like the problem we're dealing with is a little different.